articles/virtual-machines/windows/windows-desktop-multitenant-hosting-deployment.md

```diff
@@ -29,13 +29,14 @@ For more details about subscription licenses that qualify to run Windows 10 on A
 > [!IMPORTANT]
 > Users **must** have one of the below subscription licenses in order to use Windows 10 images in Azure for any production workload. If you do not have one of these subscription licenses, they can be purchased through your [Cloud Service Partner](https://azure.microsoft.com/overview/choosing-a-cloud-service-provider/) or directly through [Microsoft](https://www.microsoft.com/microsoft-365?rtc=1).
 
-**Eligible subscription licenses:**
+## Operating systems and licenses
 
--	Windows 10 Enterprise E3/E5
--	Windows 10 Education A3/A5 
--	Microsoft 365 A3/A5
--	Microsoft 365 F3/E3/E5 
--	Windows VDA E3/E5
+You have a choice of operating systems that you can use for session hosts to provide virtual desktops and remote apps. You can use different operating systems with different host pools to provide flexibility to your users. Supported dates are inline with the [Microsoft Lifecycle Policy](/lifecycle/). We support the following 64-bit versions of these operating systems: 
+
+|Operating system |User access rights|
+|---|---|
+|<ul><li>Windows 11 Enterprise multi-session</li><li>Windows 11 Enterprise</li><li>Windows 10 Enterprise multi-session, version 1909 and later</li><li>Windows 10 Enterprise, version 1909 and later</li><li>Windows 7 Enterprise</li></ul>|License entitlement:<ul><li>Microsoft 365 E3, E5, A3, A5, F3, Business Premium, Student Use Benefit</li><li>Windows Enterprise E3, E5</li><li>Windows VDA E3, E5</li><li>Windows Education A3, A5</li></ul>External users can use [per-user access pricing](https://azure.microsoft.com/pricing/details/virtual-desktop/) instead of license entitlement.</li></ul>|
```

Comment:
Can we please remove Windows 10 Enterprise multi-session, version 1909 and later as this is only for Azure Virtual Desktop and not IaaS VMs. Also, remove Windows 7 as we have another doc on that already and it is only supported for dev/test.

Reply:
Edited.